src/lib/utils/thorchain/hooks/useSendThorTx.tsx

```diff
@@ -270,9 +271,6 @@ export const useSendThorTx = ({
     if (accountNumber === undefined) return
     if (isToken(fromAssetId(asset.assetId).assetReference) && !inboundAddressData) return
 
-    if (!shouldUseDustAmount && !bn(amountOrDustCryptoBaseUnit).gt(0))
-      throw new Error('invalid amount specified')
-
```

Comment:
Sending a 0 amount should be valid for sure (this is the case for RUNEPool withdraws)

Reply:
blocking: we *do* need this for savers and LP withdraws, see `shouldUseDustAmount` above @NeOMakinG:

https://github.com/shapeshift/web/blob/9a5e3a5b7b41e2d9de29aa581d05f09d41889a99/src/lib/utils/thorchain/hooks/useSendThorTx.tsx#L93-L95


The condition should stay, but we should check for runepool withdraws. Removing this line means:

1. we wouldn't check for valid deposit `value` gt(0) for deposit-like actions (borrow, savers, LP and RUNEPool deposits)
2. we wouldn't check for valid deposit `dust` gt(0) for savers and LP withdraws

```
+     if (action !== 'withdrawRunepool` && !shouldUseDustAmount && !bn(amountOrDustCryptoBaseUnit).gt(0))
+       throw new Error('invalid amount specified')
+  
```